lib/src/rules/avoid_web_libraries_in_flutter.dart

```diff
@@ -2,13 +2,15 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
+import 'dart:io';
```

Comment:
I can't tell where this is being used, but it's somewhat concerning if we really need it.